src/app_service/service/community/dto/community.nim

```diff
@@ -95,6 +96,16 @@ proc toChat*(jsonObj: JsonNode): Chat =
   discard jsonObj.getProp("position", result.position)
   discard jsonObj.getProp("categoryID", result.categoryId)
 
+proc toChat*(chat: ChatDto): Chat =
```

Comment:
aaah okay, I just didn't want to make it more confusing. Isn't it better to get rid of the Chat here and replace it with the ChatDto instead. these two seem confusing

Reply:
Maybe in a separate task?

Reply:
> Isn't it better to get rid of the Chat here and replace it with the ChatDto instead. these two seem confusing

It could be a bit confusing yes, but look at that like `chats` from `CommunityDto` should contain some light info about chats that community has and `ChatDto` contains more details about it. 
If you use `ChatDto` instead of `Chat` that's ok but, but in that case you will need a method `chatToChatDto` in order to use appropriate type for `SIGNAL_COMMUNITY_CHANNEL_CREATED` :)

So maybe would be better to follow this [comment](https://github.com/status-im/status-desktop/pull/4511#discussion_r789482784), but you decide.

> Maybe in a separate task?

No need, that should be a small change, just moving code you already have around.

Reply:
Have done it as per your comment. but we should think about a better way, maybe status_go needs to be streamlined, the processing to understand what has changed and updating it isn't good